lib/editor/components/ExceptionDateRange.js

```diff
@@ -15,14 +15,23 @@ type Props = {
   activeComponent: string,
   activeEntity: ScheduleException,
   index: number,
-  isInvalid: boolean,
   range: ScheduleExceptionDateRange,
-  updateActiveGtfsEntity: typeof updateActiveGtfsEntity
+  updateActiveGtfsEntity: typeof updateActiveGtfsEntity,
+  validationState: string
 }
 
-export default class ExceptionDateRange extends Component<Props> {
+type State = {
+  validationStateOverride: boolean
```

Comment:
Is this for forcing an error per the `render` method? Maybe rename to `forceErrorState`

Reply:
Yep, I like the rename. It's updated as of https://github.com/ibi-group/datatools-ui/pull/828/commits/bd59f94bd41bd062d40630c054cacfdc97f32a98